templates/projects/tasks_browse.webpack.ejs

```diff
@@ -240,6 +240,8 @@
                         <a class="label label-warning" target="_blank" href="{{ url_for('project.task_presenter', short_name=project.short_name, task_id=t.id, mode='read_only') }}">#{{ t.id }}</a>
                     {% elif can_know_task_is_gold and task_pct >= 100 %}
                         <a class="label label-success" target="_blank" href="{{ url_for('project.task_presenter', short_name=project.short_name, task_id=t.id, mode='read_only') }}">#{{ t.id }}</a>
+                    {% elif not ((current_user.subadmin and current_user.id in project.owners_ids) or current_user.admin) %}
+                        <a class="label label-info" target="_blank" href="{{ url_for('project.task_presenter', short_name=project.short_name, task_id=t.id, mode='cherry_pick') }}">#{{ t.id }}</a>
```

Comment:
is there any concern passing `mode=cherry_pick` from the UI?

Reply:
cherry_pick mode is to be decided by server and to be set based on condition scheduler= task_queue, no? Is there any complexity in achieving this?

Reply:
"mode" is passed along by request.args, which is an immutable objects so cannot be overwritten by server

Reply:
Can we have an explicit server-side check to ensure the user is eligible?

Reply:
[Here](https://github.com/bloomberg/pybossa/blob/master/pybossa/view/projects.py#L1545) we know the scheduler type. If we pass the scheduler type under project object, i think it will be available here. What do you think?